src/index.scss

```diff
@@ -36,16 +33,15 @@ $white: #fff;
       white-space: nowrap;
       overflow: hidden;
       text-overflow: ellipsis;
-      padding-bottom: 0.1rem;
+      padding-bottom: calc(var(--pgn-spacing-spacer-base)* 0.1);
     }
   }
 
-  .user-dropdown {
-    .btn {
-      height: 3rem;
-      // @media (max-width: -1 + map-get($grid-breakpoints, "sm")) {
-      //   padding: 0 0.5rem;
-      // }
+  .user-dropdown .btn {
+    height: 3rem;
+
+    @media (--max-pgn-size-breakpoint-xs) {
```

Comment:
@dcoa: in https://github.com/openedx/frontend-component-header/pull/351#issuecomment-2719232736 you mentioned
> I could not find a way to deal with media screen

It looks like this is the only place where we have `@media` using `@custom-media` in this PR, and it's replacing code where the media query is commented out. Unless there's something I'm overlooking, it seems to me like we should just be able to remove this media query entirely.